iphone/Maps/Classes/CustomAlert/Toast/Toast.swift

```diff
@@ -47,26 +44,29 @@ final class Toast: NSObject {
   }
 
   @objc func show(in view: UIView?, alignment: Alignment) {
-    guard let v = view else { return }
+    guard let view else { return }
     blurView.translatesAutoresizingMaskIntoConstraints = false
-    v.addSubview(blurView)
-    let views = ["bv" : blurView]
-    v.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|->=16-[bv]->=16-|",
-                                                    options: [],
-                                                    metrics: [:],
-                                                    views: views))
-    let formatString = alignment == .bottom ? "V:[bv]-50-|" : "V:|-50-[bv]"
-    v.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: formatString,
-                                                    options: [],
-                                                    metrics: [:],
-                                                    views: views))
-    v.addConstraint(NSLayoutConstraint(item: blurView,
-                                       attribute: .centerX,
-                                       relatedBy: .equal,
-                                       toItem: v,
-                                       attribute: .centerX,
-                                       multiplier: 1,
-                                       constant: 0))
+    view.addSubview(blurView)
+    
+    let leadingConstraint = blurView.leadingAnchor.constraint(greaterThanOrEqualTo: view.leadingAnchor, constant: 16)
+    let trailingConstraint = blurView.trailingAnchor.constraint(lessThanOrEqualTo: view.trailingAnchor, constant: -16)
+    
+    NSLayoutConstraint.activate([
+      leadingConstraint,
+      trailingConstraint
+    ])
+    
+    let topConstraint: NSLayoutConstraint
+    if alignment == .bottom {
+      topConstraint = blurView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -63)
```

Comment:
nit: it would be great to have some comments about how these constants were calculated in the future. If comments are short, they can be converted to a named constant.